python/ml4ir/applications/ranking/tests/test_lr_schedule.py

```diff
@@ -0,0 +1,217 @@
+import sys; sys.path.insert(0, '/Users/mohamed.m/Documents/work/projects/ml4ir/python/')
```

Comment:
This is a red flag.. 
I am assuming it also means that you have not properly installed ml4ir in your local? You should try to follow the README, if it fails we should sync.

Reply:
Oh this is my bad. I included the wrong file. Nice catch.